src-vue/src/views/ModsView.vue

```diff
@@ -4,9 +4,11 @@
             <h3>Installed Mods:</h3>
             <div>
                 <el-card shadow="hover" v-for="mod in $store.state.installed_mods">
-                    <el-switch style="--el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5" v-model="mod.enabled"
-                        :before-change="() => updateWhichModsEnabled(mod)" :loading="global_load_indicator" />
-                    {{mod.name}}
+                    <el-switch style="--el-switch-on-color: #13ce66; --el-switch-off-color: #8957e5"
+                        v-model="mod.enabled" :before-change="() => updateWhichModsEnabled(mod)"
+                        :loading="global_load_indicator" />
+                    <el-button type="danger" @click="deleteMod(mod)">Delete</el-button>
+                    {{ mod.name }}
```

Comment:
Rendered it looks like this which could definitely be improved :P

![image](https://user-images.githubusercontent.com/40122905/205782214-c993a314-fe7b-40eb-aa21-751a130ef25b.png)

Reply:
I assume this will done after this get merged? 😄

Reply:
I would like users not to delete their mods accidentally, so we should ask for their confirmation before effectively deleting mods: https://element-plus.org/en-US/component/popconfirm.html

Reply:
I guess. Unless you want to tack it on to this PR? :eyes:

Reply:
I would like to reuse the card layout from Thunderstore view, but I still gotta figure out how :)

Reply:
Yeah, we might have to do some UI prototyping for ModsView ^^

Reply:
Kinda out of scope for this then